tests/dapr_tests.mk

```diff
@@ -90,6 +90,10 @@ setup-test-env-kafka:
 # Install redis and kafka to test cluster
 setup-test-env: setup-test-env-kafka setup-test-env-redis
 
+# Apply default config yaml
+setup-test-config:
+	$(KUBECTL) apply -f ./tests/config/dapr_default_config.yaml --namespace $(DAPR_TEST_NAMESPACE)
```

Comment:
I would rename dapr_default_config.yaml to dapr_mtls_off_config.yaml to be more specific.

Reply:
Sure